src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,15 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[] {
+            lottery.getRandomBall(),
+            lottery.getRandomBall(),
+            lottery.getRandomBall()
+        };
+
+        for (Ball ball : balls) {
+            System.out.println(ball);
+        }
```

Comment:
rewrite this solution without using an array,
just call needed method once - in a loop